mitosheet/src/components/taskpanes/Graph/ColumnCard.tsx

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Saga Inc.
+
+import React from 'react';
+import XIcon from '../../icons/XIcon';
+import Select from '../../elements/Select';
+import Row from '../../spacing/Row';
+import Col from '../../spacing/Col';
+
+import '../../../../css/taskpanes/Graph/AxisSection.css'
+import { ColumnID, ColumnIDsMap } from '../../../types';
+import DropdownItem from '../../elements/DropdownItem';
+import { getDisplayColumnHeader } from '../../../utils/columnHeaders';
+
+const ColumnCard = (props: {
```

Comment:
Leave a comment that this is just a select an an X-Icon.

Also, can we refactor this out of graph -> a component it's own right? And then use it for Pivot for the row and column sections?

In fact, I'm wondering if we already have a component for that here.